ArduCopter/Parameters.cpp

```diff
@@ -30,6 +30,12 @@
 #endif
 
 const AP_Param::Info Copter::var_info[] = {
+
+     // @Param: ACL_LIMIT
```

Comment:
Indent

Reply:
Units missing

Reply:
Display name incorrect

Reply:
I am sure @rmackay9 will want a different name for this param...accelerations are always 'ACC'